.github/ISSUE_TEMPLATE/01-bug-report-plugin.yaml

```diff
@@ -1,13 +1,13 @@
 name: '🐛 Rule Bug'
-description: 'Report a bug you encountered with a lint rule'
+description: 'An issue with a single lint rule'
 title: 'Bug: [rule name here] <short description of the issue>'
 labels:
   - bug
   - 'package: eslint-plugin'
   - triage
 body:
   - type: checkboxes
-    id: sanity-checks
+    id: preliminary-checks
```

Comment:
A find-and-replace here: "sanity" has meaning around mental health that I've been avoiding. "preliminary" is a bit more precise in meaning here.